HomeWorkApp4.java

```diff
@@ -0,0 +1,135 @@
+import java.util.Arrays;
+
+public class HomeWorkApp4 {
+
+    public static void main(String[] args) {
+        //System.out.println("Задание №1:");
+        //replacementArray(new int[] {1,0,1,0,0});
+
+
+        //System.out.println("Задание №2:");
+        //countArray();
+
+        //System.out.println("Задание №3:");
+        //int[] arr = {1, 5, 3, 2, 11, 4, 5, 2, 4, 8, 9, 1 };
+        //System.out.println(Arrays.toString(multiplicationArray(arr)));
+
+        //System.out.println("Задание №4.1:");
+        //mainDiagonalArray(new int[10][10]);
+        System.out.println("Задание 4.2:");
+        maiAdditionDiagonalArray(new int[10][10]);
+
+        //System.out.println("Задание №5:");
+        //System.out.println(Arrays.toString(createArray(10,7)));
+
+        //System.out.println("Задание №6:");
+        //int[] arr = {1, 5, 3, 2, 11, 4, 5, 2, 4, 8, 9, -100 };
+        //minMaxValue(arr);
+
+        //System.out.println("Задание №7:");
+        //int[] arr = {1, 1, 1, 2, 1, 2 };
+        //System.out.println(isBalance(arr));
+
+
+
+
+    }
+
+    public static void replacementArray(int[] arr){
+        for (int i = 0; i < arr.length; i++) {
+            //arr[i] = arr[i] == 0 ? 1 : 0;
+            if (arr[i]==0){
+                arr[i]=1;
+            } else { arr[i]=0;}
```

Comment:
Надо придерживаться одного форматирования. Мешать форматы не лучшая идея. Почитай Java Code Convention, там много об этом сказано. Дефакто международный стандарт
```suggestion
            if (arr[i]==0){
                arr[i]=1;
            } else { 
                arr[i]=0;
            }
```